client/landing/stepper/declarative-flow/internals/steps-repository/pattern-assembler/screen-category-list.tsx

```diff
@@ -97,7 +97,7 @@ const ScreenCategoryList = ( {
 							role="option"
 							as="button"
 							{ ...composite }
-							className={ classNames( 'screen-category-list__category-button navigator-button', {
+							className={ classNames( 'screen-category-list__category-button', {
```

Comment:
The category list panel seems broken after `navigator-button` was removed:
![Screenshot 2023-06-26 at 3 51 43 PM](https://github.com/Automattic/wp-calypso/assets/797888/49ebf327-bf25-4957-a941-3f22c009aa43)